Makefile-test.mk

```diff
@@ -68,7 +71,8 @@ test-integration: gomod-download envtest ginkgo mpi-operator-crd ray-operator-cr
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
 	KUEUE_BIN=$(PROJECT_DIR)/bin \
 	ENVTEST_K8S_VERSION=$(ENVTEST_K8S_VERSION) \
-	$(GINKGO) $(GINKGO_ARGS) -procs=$(INTEGRATION_NPROCS) --race --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
+	API_LOG_LEVEL=$(INTEGRATION_API_LOG_LEVEL) \
+	$(GINKGO) $(GINKGO_ARGS) -procs=$(INTEGRATION_NPROCS) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
```

Comment:
I would suggest not to remove the `race` in this PR, it is not contributing to "Add API Logging".

Reply:
sure, done